cluster-autoscaler/cloudprovider/ionoscloud/ionoscloud_manager.go

```diff
@@ -26,14 +26,15 @@ import (
 	"github.com/google/uuid"
 	apiv1 "k8s.io/api/core/v1"
 	"k8s.io/autoscaler/cluster-autoscaler/cloudprovider"
+	ionos "k8s.io/autoscaler/cluster-autoscaler/cloudprovider/ionoscloud/ionos-cloud-sdk-go"
 	"k8s.io/klog/v2"
 )
 
 const (
 	envKeyClusterId    = "IONOS_CLUSTER_ID"
-	envKeyToken        = "IONOS_TOKEN"
+	envKeyToken        = ionos.IonosTokenEnvVar
+	envKeyEndpoint     = ionos.IonosApiUrlEnvVar
 	envKeyInsecure     = "IONOS_INSECURE"
-	envKeyEndpoint     = "IONOS_ENDPOINT"
```

Comment:
The endpoint env var name change is breaking. Are you aware of a place where we need to respect that?

Reply:
No, it's only used by us anyways